model/config.go

```diff
@@ -589,7 +589,7 @@ func (o *Config) SetDefaults() {
 		*o.LocalizationSettings.DefaultClientLocale = DEFAULT_LOCALE
 	}
 
-	if o.LocalizationSettings.AvailableLocales == nil {
+	if o.LocalizationSettings.AvailableLocales == nil || *o.LocalizationSettings.AvailableLocales == "" {
```

Comment:
If we're setting it here, won't it still get written back to the config file when the user saves? Then we'll still have the same problem that we're having now